samples/bynder-dialog/src/BynderDialog.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Dialog extends React.Component {
+  static propTypes = {
+    sdk: PropTypes.object.isRequired,
+  };
+
+  componentDidMount() {
+    this.loadBynderScript();
+    this.props.sdk.window.updateHeight();
+    document.addEventListener('BynderAddMedia', e => {
+      const assets = e.detail.map(asset => ({
+        id: asset.id,
+        src: asset.thumbnails['webimage'],
+      }));
+      this.props.sdk.close(assets || []);
```

Comment:
Default value is redundant here, if we want to be safe I'd suggest to check whether the event actually included an array:
```
      const eventDetail = Array.isArray(e.detail) ? e.detail : [];
      const assets = eventDetail.map(asset => ({
        id: asset.id,
        src: asset.thumbnails['webimage'],
      }));
      this.props.sdk.close(assets);
```